docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
+
+## About
+
+<!--
+Matryca testowa służy do określenia które z zetsawów i przypadków testych
+opisanych w dokumentacji testowej dedykowane są dla danej platformy
+-->
+
+## Test matrix - ASUS KGPE-D16
+
+### Module: dasharo-compatibility
+
+| No.  | Supported test suite                              | Supported test cases                 |
+|:----:|:-------------------------------------------------:|:------------------------------------:|
+| 1.   | coreboot base port                                | All                                  |
+| 2.   | SMBIOS                                            | DMI002.001, DMI003.001               |
```

Comment:
Let's link specific tests

Reply:
When someone will read that first question would be: why other test cases from that suite are not supported? Isn't it way better to tell what is not supported and why? Of course it may depend on our goal for this document, but I'm not sure I understand the goal now.